cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -309,8 +296,14 @@ def start_kube_dns():
             'dns_domain': hookenv.config('dns_domain')
         }
     }
-    create_addon('kubedns-controller.yaml', context)
-    create_addon('kubedns-svc.yaml', context)
+
+    try:
+        create_addon('kubedns-controller.yaml', context)
+        create_addon('kubedns-svc.yaml', context)
+    except CalledProcessError:
+        hookenv.log('kubedns deploy failed, will try again soon')
```

Comment:
would it be prudent to notify via status that addon updates are failing atm? Open question, not a blocker. I like whats here.